cmd/nvidia-ctk/runtime/configure/configure.go

```diff
@@ -46,6 +46,9 @@ const (
 	defaultCrioConfigFilePath       = "/etc/crio/crio.conf"
 	defaultDockerConfigFilePath     = "/etc/docker/daemon.json"
 
+	defaultContainerdDropInConfigFilePath = "/etc/containerd/config.d/99-nvidia.toml"
```

Comment:
Nit: Did we want to use the `/run/nvidia/{{ .something }}` here or are we _hoping_ that we could convince Containerd to start honoring this path?

Reply:
Since this is for interactive usage, I thought it was ok to set it like that, and yes, with the Hope that we can push the community towards this direction in the future.

Reply:
OK. That's fine then. Let's also check the paths we specify in the GPU Operator as part of https://github.com/NVIDIA/gpu-operator/pull/1710 as well.

Reply:
The GPU Operator is using the right default 

https://github.com/NVIDIA/gpu-operator/blob/1dda24966ee5e793a5eb4aa1f7abe477ef410f3c/controllers/object_controls.go#L62-L63